js/script.js

```diff
@@ -23,4 +24,107 @@ document.addEventListener('DOMContentLoaded', () => {
+      linklive: '#',
+      linksource: '#',
+    },
+  ];
+  const worksgrid = document.getElementById('works-grid');
+  let popupblock = `<div id="container-exist" class="project-container"><div class="project-text-section"><div class="popup-heading"><h2 class="works-heading">${works[i].projectname}</h2><a href="#" class="popup-cancel" id="closeproject"><img src="./images/icons/desktop-popup-cancel.png"></a></div><div class="works-frame"><label class="frame-text1">Canopy</label><div class="frame-counter"><img src="images/Counter.png"></div><label class="frame-text2">Back End Dev</label><div class="frame-counter"><img src="images/Counter.png"></div><label class="frame-text2">2015</label></div><div class="img-container"><img class ="mobile-popup-image" src="${works[i].featuredimage}"></div><div class="img-container-desktop-popup"><img class="desktop-image" src="${works[i].featuredimagepopup}"></div></div><div class="popup-desktop-format"><p class="works-paragraph">${works[i].detaileddescription}</p><div class="popup-desktop-format-2"><ul class="works-list-popup"><li><label class="list-buttons">${works[i].technologies[0]}</label></li><li><label class="list-buttons">${works[i].technologies[1]}</label></li><li><label class="list-buttons">${works[i].technologies[2]}</label></li></ul><div class="popup-divider"></div><div class="project-button-section"><a href="${works[i].linklive}"class="works-button-1">See Live <img src="./images/icons/See-livve-icon.png"></img><a href="${works[i].linksource}"class="works-button-1">See Source <img src="./images/icons/github-button.png"></img></div></div></div><div class="navigation-links"><a class="popup-links" id="previous" href="#">Previous Project</a><a class="popup-links" id="next" href="#">Next Project</a></div></div>`;
+  const worksgrid1 = document.getElementById('works-grid');
+  for (let j = 0; j < works.length; j += 1) {
+    const codeblock = `<div id="works-1"> <div class="img-container"><img src="${works[j].featuredimage}"></div><div class="img-container-desktop"><img class="desktop-image" src="${works[j].featuredimagedesktop}"></div> <div class="works-contents"> <h2 class="works-heading">${works[j].projectname}</h2> <div class="works-frame"> <label class="frame-text1">Canopy</label> <div class="frame-counter"><img src="images/Counter.png"></div><label class="frame-text2">Back End Dev</label><div class="frame-counter"><img src="images/Counter.png"></div><label class="frame-text2">2015</label></div><p class="works-paragraph">${works[j].projectdescription}</p><ul class="works-list"><li><label class="list-buttons">${works[j].technologies[0]}</label></li><li><label class="list-buttons">${works[j].technologies[1]}</label></li><li><label class="list-buttons">${works[j].technologies[2]}</label></li></ul><button id="project${j}"class="works-button">See Project</button></div></div>`;
+    worksgrid.innerHTML += codeblock;
+  }
+
+  for (let j = 0; j < works.length; j += 1) {
```

Comment:
- [ ] Nice work making the requested changes :clap: However the project image in the popup (mobile version and desktop version) is small compare to the Figma Design(see image). Kindly chect it to follow Figma's Design as requested in the project requirement.

###Your Design
![Screenshot from 2023-04-12 15-28-58](https://user-images.githubusercontent.com/91142432/231508339-467df690-5acf-458e-8580-242a6ac64926.png)

### What it's supposed to look like:
![Screenshot from 2023-04-12 15-24-10](https://user-images.githubusercontent.com/91142432/231508445-423cadb2-37d2-4fd6-9a40-29e82e163260.png)

Reply:
@LionRogue1 Hi, Thank you. As mentioned above. The Figma view size is square with a way higher height than a typical 16:9 window. I understand your concern. but it won't be possible without putting a huge overflow in the y direction. Please consider that as I have mentioned above too. The viewport size for the Figma is 1440 x 1577. While on a 16:9 screen, it would translate into 1440 x 810. No way a box of this size would fit in even half of it. Without the use of overflow. Honestly, I don't prefer using that but I had to regardless because of minor glitches. Kindly reconsider this. I am not understanding this at all.

Reply:
> 
@LionRogue1 Hi, Thank you. As mentioned above. The Figma view size is square with a way higher height than a typical 16:9 window. I understand your concern. but it won't be possible without putting a huge overflow in the y direction. Please consider that as I have mentioned above too. The viewport size for the Figma is 1440 x 1577. While on a 16:9 screen, it would translate into 1440 x 810. No way a box of this size would fit in even half of it. Without the use of overflow. Honestly, I don't prefer using that but I had to regardless because of minor glitches. Kindly reconsider this. I am not understanding this at all.